docs/source/Usage.md

```diff
@@ -168,6 +168,7 @@ The following is the list of the available analyzers you can run out-of-the-box.
 * `PEframe_Scan`: Perform static analysis on Portable Executable malware and malicious MS Office documents with [PeFrame](https://github.com/guelfoweb/peframe)
 * `Cymru_Hash_Registry_Get_File`: Check if a particular file is known to be malware by [Team Cymru](https://team-cymru.com/community-services/mhr/)
 * `Thug_HTML_Info`: Perform hybrid dynamic/static analysis on a HTML file using [Thug low-interaction honeyclient](https://thug-honeyclient.readthedocs.io/)
+* `cape_sandbox`: [CAPESandbox](https://capesandbox.com) automatically scans suspicious files using the CapeSandbox API.
```

Comment:
the name is different from the one in the `analyzer_config.json`. We could explicitly say that this analyzer can be used for private instances too.

Reply:
Noted.